pyartcd/pyartcd/pipelines/rebuild_golang_rpms.py

```diff
@@ -207,6 +190,31 @@ def get_rpms(self, el_v):
         rpms = [b for b in latest_builds if not ('-container' in b or b.startswith('rhcos-'))]
         return rpms
 
+    @staticmethod
+    # Squash the Release segment to it's significant number and bump
+    # 42.rhaos4_8.el8 -> 43.rhaos4_8.el8
+    # 42.1.rhaos4_8.el8 -> 43.rhaos4_8.el8
+    # rhaos4_8.el8 -> 1.rhaos4_8.el8
+    def bump_release(release):
+        k = release.split('.')
+        numbers = []
+        non_number_marker = None
+        for p in k:
+            if p.isdigit():
+                numbers.append(p)
+            else:
+                non_number_marker = release.index(p)
+                break
+
+        if len(numbers) < 1:
+            rel = "1"
+        else:
+            rel = f'{int(numbers[0]) + 1}'
+
+        rel += f'.{release[non_number_marker:]}'
```

Comment:
non_number_marker can be none

Reply:
applied now:
        if non_number_marker is not None:
            rel += f'.{release[non_number_marker:]}'